test/ci-stack.test.ts

```diff
@@ -112,3 +112,12 @@ test('LoadBalancer', () => {
     ],
   }, ResourcePart.Properties));
 });
+
+test('Stack with macAgent', () => {
+  const app = new App({
+    context: { useSsl: 'true', runWithOidc: 'true', macAgent: true },
+  });
+
+  // WHEN
+  new CIStack(app, 'MyTestStack', {});
+});
```

Comment:
Did you miss the `//THEN`, not sure what we are testing here.

Reply:
Here //THEN might not able to check the contents of the Jenkins JCASC yaml @gaiksaya hence just added `new CIStack(app, 'MyTestStack', {});`.

Reply:
I think then its irrelevant to add this as well. Also one way to check if the JCASC is like this https://github.com/opensearch-project/opensearch-ci/blob/main/test/compute/oidc-config.test.ts#L38-L42
The newly generated jenkins.yaml should contain your config

Reply:
Hey @gaiksaya I have updated the tests in my latest commit, please check.

Reply:
Can you remove this test then? As we are already creating stack in below test
https://github.com/opensearch-project/opensearch-ci/blob/a2bb132ddf4dd1f964a89a6bbc75da59d22d5596/test/compute/agent-node-config.test.ts#L22

Reply:
done @gaiksaya